go.mod

```diff
@@ -3,11 +3,12 @@ module github.com/openyurtio/raven-controller-manager
 go 1.15
 
 require (
-	github.com/go-logr/logr v0.3.0
+	github.com/go-logr/logr v1.2.0
+	github.com/projectcalico/api v0.0.0-20220505235232-ce7a5122e146
 	github.com/stretchr/testify v1.7.1
-	k8s.io/api v0.19.7
-	k8s.io/apimachinery v0.19.7
-	k8s.io/client-go v0.19.2
-	k8s.io/klog/v2 v2.2.0
-	sigs.k8s.io/controller-runtime v0.7.2
+	k8s.io/api v0.23.2
```

Comment:
The latest kubernetes version that OpenYurt currently supports is 1.22 and most of the OpenYurt components use v0.22.3 as kubernetes library version. 

So how about downgrading the version to v0.22.3.

Reply:
The latest code has been commited. `k8s components` have been downgraded to v0.22.3.

Reply:
@luckymrwang It is still v0.23.2, maybe you missed the change.

Reply:
github.com/projectcalico/api@v0.0.0-20220505235232-ce7a5122e146  requires k8s.io/api@v0.23.2 
repalce v0.22.3 to v0.23.2 will generate `rest.HTTPClientFor` in clientset (`make generate`) which is not support in v0.22.3